H/highs/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "highs"
+version = v"0.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ERGO-Code/HiGHS.git", "30de0e3c2c977bd3c5bfd95e502c2ee041aa6525"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+mkdir -p HiGHS/build
+cd HiGHS/build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+make
```

Comment:
Is it possible to enable parallel compilation?
```suggestion
make -j${nproc}
```
It's not a too big deal though

Reply:
I'll try locally

Reply:
seems good yup